cognite/client/utils/_azure_token_generation.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from cognite.client.exceptions import CogniteAPIKeyError
+from cognite.client.utils import _auxiliary
+
+
+def _azure_environment_vars_set():
+    azure_api_client_id = os.getenv("AZURE_API_CLIENT_ID")
+    azure_api_client_secret = os.getenv("AZURE_API_CLIENT_SECRET")
+    oidc_authority = os.getenv("OIDC_AUTHORITY")
```

Comment:
This is very Azure AD specific (not Azure, but Azure AD). We'll want to make this more general 2021Q1..